2023-11-25 02:36:03.884 UTC [375271] LOG:  number of prepared transactions has not been configured, overriding
2023-11-25 02:36:03.884 UTC [375271] DETAIL:  max_prepared_transactions is now set to 200
2023-11-25 02:36:03.902 UTC [375271] LOG:  starting PostgreSQL 15.3 on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 11.3.0-1ubuntu1~22.04.1) 11.3.0, 64-bit
2023-11-25 02:36:03.902 UTC [375271] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2023-11-25 02:36:03.911 UTC [375271] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2023-11-25 02:36:03.919 UTC [375274] LOG:  database system was shut down at 2023-11-25 02:36:03 UTC
2023-11-25 02:36:03.927 UTC [375271] LOG:  database system is ready to accept connections
2023-11-25 02:37:56.485 UTC [392096] LOG:  starting maintenance daemon on database 16427 user 10
2023-11-25 02:37:56.485 UTC [392096] CONTEXT:  Citus maintenance daemon for database 16427 user 10
2023-11-25 02:37:56.650 UTC [392100] ERROR:  could not create unique index "idx_badindex_n"
2023-11-25 02:37:56.650 UTC [392100] DETAIL:  Key (n)=(10) is duplicated.
2023-11-25 02:37:56.650 UTC [392100] STATEMENT:  CREATE UNIQUE INDEX CONCURRENTLY idx_badindex_n ON tbl_badindex (n);
2023-11-25 02:37:59.395 UTC [392273] ERROR:  permission denied for schema repack at character 8
2023-11-25 02:37:59.395 UTC [392273] STATEMENT:  select repack.version(), repack.version_sql()
2023-11-25 02:38:00.386 UTC [392362] ERROR:  table name not found for OID 1
2023-11-25 02:38:00.386 UTC [392362] STATEMENT:  SELECT repack.get_order_by('issue321_idx'::regclass::oid, 1);
2023-11-25 02:38:00.387 UTC [392362] ERROR:  cache lookup failed for index 1
2023-11-25 02:38:00.387 UTC [392362] STATEMENT:  SELECT repack.get_order_by(1, 1);
